pkg/internal/tracking/query_test.go

```diff
@@ -3,12 +3,48 @@ package tracking
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/stretchr/testify/assert"
 	"testing"
```

Comment:
Reviewdog. You should be able to set up your IDE to run `goimports` on each save, it's very helpful.

Reply:
I think I do have it, but I have to be in this file to run it. I'll see if I can setup some pre push or pre commit job that would run pre-push